docs/src/Tooltip.doc.js

```diff
@@ -1,165 +1,860 @@
+                      alt="Tropic greens: The taste of Petrol and Porcelain | Interior design, Vintage Sets and Unique Pieces agave"
+                      color="rgb(231, 186, 176)"
+                      naturalHeight={751}
+                      naturalWidth={564}
+                      src="https://i.ibb.co/7bQQYkX/stock2.jpg"
+                    >
+                      <Box padding={3}>
+                        <Heading
+                          align={alignText}
+                          color="white"
+                          size="lg"
+                        >
+                          Tropic greens: The taste of Petrol and Porcelain
+                        </Heading>
+                      </Box>
+                    </Image>
+                  </Box>
+                  <Flex direction="column" gap={4}>
+                    <Heading size="sm" weight="bold">Text Overlay</Heading>
+                    <Text size="lg">Add text directly onto your Pin</Text>
+                    <Text size="lg" weight="bold">Alignment</Text>
+                    <Flex>
+                      <Tooltip text="Align left" zIndex={TOOLTIP_Z_INDEX}>
+                        <IconButton
+                          accessibilityLabel="Align left"
+                          bgColor="white"
+                          icon="text-align-left"
+                          iconColor="darkGray"
+                          onClick={() => setAlignText('left')}
+                          size="lg"
+                          selected={alignText === 'left'}
+                        />
+                      </Tooltip>
+                      <Tooltip text="Align center" zIndex={TOOLTIP_Z_INDEX}>
+                        <IconButton
+                          accessibilityLabel="Align center"
+                          bgColor="white"
+                          icon="text-align-center"
+                          iconColor="darkGray"
+                          onClick={() => setAlignText('center')}
+                          size="lg"
+                          selected={alignText === 'center'}
+                        />
+                      </Tooltip>
+                      <Tooltip text="Align right" zIndex={TOOLTIP_Z_INDEX}>
+                        <IconButton
+                          accessibilityLabel="Align right"
+                          bgColor="white"
+                          icon="text-align-right"
+                          iconColor="darkGray"
+                          onClick={() => setAlignText('right')}
+                          size="lg"
+                          selected={alignText === 'right'}
+                        />
+                      </Tooltip>
+                    </Flex>
+                  </Flex>
+              </Box>
+            </ScrollableContainer>
+          </Modal>
+        </Layer>
+      )}
+    </>
+  )
+}`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="With ScrollableContainer"
```

Comment:
we automatically sentence case these, so it will always appear `With scrollablecontainer` one option could be "Within scrolling containers" or we'll need to update the component logic

Reply:
DONE